SConstruct

```diff
@@ -122,6 +122,12 @@ def set_defaults(env):
               default=False,
               help='Preprocess selected files for profiling')
 
+    AddOption('--with-fault-injection',
```

Comment:
Let's use 

if env["BUILD_TYPE"] != 'release'

Reply:
This should be a variable rather than an option so it is sticky.  If I'm disabling fault injection in my local build, I will want it to persist through rebuilds until I change it explicitly.